crates/ruff_python_parser/src/lib.rs

```diff
@@ -388,9 +387,8 @@ impl Tokens {
         let end = *self.first_unknown_or_len.get_or_init(|| {
             self.raw
                 .iter()
-                .find_position(|token| token.kind() == TokenKind::Unknown)
-                .map(|(idx, _)| idx)
-                .unwrap_or_else(|| self.raw.len())
+                .position(|token| token.kind() == TokenKind::Unknown)
+                .unwrap_or(self.raw.len())
         });
```

Comment:
I think this is on me as I didn't realize that `find_position` was coming from `itertools` 😬